crates/host/src/wasmbus/mod.rs

```diff
@@ -1454,6 +1454,16 @@ impl Host {
         Ok(Some(buf.into()))
     }
 
+    #[instrument(skip(self, payload))]
+    async fn fetch_actor(&self, actor_ref: &str) -> anyhow::Result<wasmcloud_runtime::Actor> {
```

Comment:
Could we avoid adding this helper for now? It's confusing to have `fetch_actor`, which calls `fetch_actor` and also initializes it.

There are only 2 places where we need this functionality and it's small enough that I think it's fine to inline

Reply:
Sorry there were some other changes not committed, this function is used in several places